src/deb/control/control

```diff
@@ -3,6 +3,6 @@ Version: [[version]]
 Section: java
 Priority: optional
 Architecture: all
-Depends: default-jre | java8-runtime
+Depends: default-jre | java8-runtime | plos-themes
```

Comment:
Is this unavoidable? I know the Debian packaging is already a PLOS-only thing. But I hate to see `plos-themes` mentioned in Wombat as a general pattern, especially if it would be one more thing to clean up if we ever generalize these Debian packages for public use.